sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -20,7 +20,13 @@
     return cast<pi_result>(reterr);                                            \
   }
 
-const char SupportedVersion[] = "1.1";
+#define STRING_HELPER(a) #a
+#define STRINGIZE(a,b) STRING_HELPER(a.b)
+const char SupportedVersion[] = STRINGIZE(_PI_H_VERSION_MAJOR,_PI_H_VERSION_MINOR);
```

Comment:
since the arguments are defined in pi.h why not define the string there too? And then you don't need to repeat stringizing utilities here